### ISSUE TITLE
Could not load locale file locale/fr_FR.json on load

### ISSUE THREAD

**Issue by k-jiang** [Dasfaust/GlobalMarket#128]
`Could not load locale file locale/fr_FR.json. Check your syntax` while initializing the plugin.
```
[22:17:30] [Server thread/ERROR]: [GlobalMarket] Could not load locale file locale/fr_FR.json. Check your syntax
com.google.gson.JsonSyntaxException: duplicate key: menu_creation_price_info
	at com.google.gson.internal.bind.MapTypeAdapterFactory$Adapter.read(MapTypeAdapterFactory.java:190) ~[spigot-1.11.2.jar:git-Spigot-7d78b81-e2a288c]
	at com.google.gson.internal.bind.MapTypeAdapterFactory$Adapter.read(MapTypeAdapterFactory.java:145) ~[spigot-1.11.2.jar:git-Spigot-7d78b81-e2a288c]
	at com.google.gson.internal.bind.ReflectiveTypeAdapterFactory$1.read(ReflectiveTypeAdapterFactory.java:93) ~[spigot-1.11.2.jar:git-Spigot-7d78b81-e2a288c]
	at com.google.gson.internal.bind.ReflectiveTypeAdapterFactory$Adapter.read(ReflectiveTypeAdapterFactory.java:172) ~[spigot-1.11.2.jar:git-Spigot-7d78b81-e2a288c]
	at com.google.gson.Gson.fromJson(Gson.java:803) ~[spigot-1.11.2.jar:git-Spigot-7d78b81-e2a288c]
	at com.google.gson.Gson.fromJson(Gson.java:768) ~[spigot-1.11.2.jar:git-Spigot-7d78b81-e2a288c]
	at com.google.gson.Gson.fromJson(Gson.java:717) ~[spigot-1.11.2.jar:git-Spigot-7d78b81-e2a288c]
	at com.google.gson.Gson.fromJson(Gson.java:689) ~[spigot-1.11.2.jar:git-Spigot-7d78b81-e2a288c]
	at me.dasfaust.gm.tools.LocaleHandler.init(LocaleHandler.java:55) [GlobalMarket-2.0.1.dev.jar:?]
	at me.dasfaust.gm.Core.onEnable(Core.java:69) [GlobalMarket-2.0.1.dev.jar:?]
	at org.bukkit.plugin.java.JavaPlugin.setEnabled(JavaPlugin.java:271) [spigot-1.11.2.jar:git-Spigot-7d78b81-e2a288c]
	at org.bukkit.plugin.java.JavaPluginLoader.enablePlugin(JavaPluginLoader.java:337) [spigot-1.11.2.jar:git-Spigot-7d78b81-e2a288c]
	at org.bukkit.plugin.SimplePluginManager.enablePlugin(SimplePluginManager.java:405) [spigot-1.11.2.jar:git-Spigot-7d78b81-e2a288c]
	at org.bukkit.craftbukkit.v1_11_R1.CraftServer.enablePlugin(CraftServer.java:375) [spigot-1.11.2.jar:git-Spigot-7d78b81-e2a288c]
	at org.bukkit.craftbukkit.v1_11_R1.CraftServer.enablePlugins(CraftServer.java:325) [spigot-1.11.2.jar:git-Spigot-7d78b81-e2a288c]
	at net.minecraft.server.v1_11_R1.MinecraftServer.t(MinecraftServer.java:419) [spigot-1.11.2.jar:git-Spigot-7d78b81-e2a288c]
	at net.minecraft.server.v1_11_R1.MinecraftServer.l(MinecraftServer.java:380) [spigot-1.11.2.jar:git-Spigot-7d78b81-e2a288c]
	at net.minecraft.server.v1_11_R1.MinecraftServer.a(MinecraftServer.java:335) [spigot-1.11.2.jar:git-Spigot-7d78b81-e2a288c]
	at net.minecraft.server.v1_11_R1.DedicatedServer.init(DedicatedServer.java:272) [spigot-1.11.2.jar:git-Spigot-7d78b81-e2a288c]
	at net.minecraft.server.v1_11_R1.MinecraftServer.run(MinecraftServer.java:542) [spigot-1.11.2.jar:git-Spigot-7d78b81-e2a288c]
	at java.lang.Thread.run(Unknown Source) [?:1.8.0_111]
[22:17:30] [Server thread/INFO]: [BreedingLimit] Enabling BreedingLimit v1.0
```
Related to https://github.com/Dasfaust/GlobalMarket/commit/cb62901

This issue occurred on Spigot 1.11.2 build 1129.

**Comment by Galaxias** [comment]
This will be fixed by #130 . Sorry about that. I'll let @Dasfaust  let you know when the build is available.

**Comment by Dasfaust** [comment]
Fixed with latest commit.
